rust_src/src/lisp.rs

```diff
@@ -124,6 +124,21 @@ impl<T> ExternalPtr<T> {
     pub fn replace_ptr(&mut self, ptr: *mut T) {
         self.0 = ptr;
     }
```

Comment:
As Clippy says, these should all be unsafe methods.

Reply:
Fixed.